src/qp_klp/Assays.py

```diff
@@ -478,6 +478,8 @@ def qc_reads(self):
         # a pipeline object.
         config = self.pipeline.get_software_configuration('nu-qc')
 
+        if not hasattr(self, 'files_regex'):
+            self.files_regex = 'SPP'
```

Comment:
What does this SPP regex match?  Some sort of internal files produced by a partial SPP run?